Modules/ThirdParty/Eigen3/CMakeLists.txt

```diff
@@ -12,18 +12,51 @@ set(_Eigen3_min_version 3.3)
 option(ITK_USE_SYSTEM_EIGEN "Use an outside build of Eigen3." ${ITK_USE_SYSTEM_LIBRARIES})
 mark_as_advanced(ITK_USE_SYSTEM_EIGEN)
 
-if(ITK_USE_SYSTEM_EIGEN)
-  set(_Eigen3_SYSTEM_OR_INTERNAL "Eigen3")
-else()
-  set(_Eigen3_SYSTEM_OR_INTERNAL "ITKInternalEigen3")
-endif()
+# Docs:
+# find_package(ITK REQUIRED COMPONENTS
+# ITKEigen3
+# )
+# When requiring finding this module, the following cmake is invoked:
```

Comment:
Not sure how to interpret this -- "When requiring finding this module" -> "When this module is a dependency"?

Reply:
Thanks a lot Matt.
I went for: When this module is required

Let me know if When this module is a dependency reads better.